Fix OpenSSL issue in bundled Ruby

ohwutup opened the pull request:
This fixes https://github.com/CocoaPods/CocoaPods-app/issues/280 and https://github.com/CocoaPods/CocoaPods-app/issues/271 .

orta commented:
Oh nice!

ohwutup commented:
Looks like CI is failing on downloading SVN. :confused:. I am more than happy to fix that if you don't mind two things in the PR?

orta commented:
I'm seeing it locally
```
/tmp ⏛  /usr/bin/curl -sSL https://archive.apache.org/dist/subversion/subversion-1.8.13.tar.gz -o downloads/subversion-1.8.13.tar.gz
curl: (23) Failed writing body (0 != 512)
```

Yep - we've experienced these hosts going down before, and talked about storing the downloads as a separate repo

alloy commented:
Great work 👍

The SVN issue is just a temp one, see https://archive.apache.org/dist/subversion “This service is currently undergoing maintenance!”

ohwutup commented:
Ohh, interesting. In the work that I've been doing I have changed a bit about what is getting built into the bundle (i.e. removed svn, git, and some others) I would love to know more about why you are including all those extra tools inside the bundle? It might be something that I do in fact need but am not aware of.

orta commented:
yeah, I think that's fine - I was about to say this is mergable too 

![screen shot 2016-04-08 at 20 59 37](https://cloud.githubusercontent.com/assets/49038/14395973/d3fae3e6-fdcc-11e5-8c77-ee4ab64e609f.png)

alloy commented:
CocoaPods supports downloading from SVN, Mercurial, etc. Sometimes people have older versions. The whole point of this app is to be able to use it in any env, no matter what the user does or does not have installed.

ohwutup commented:
Ahh, that makes sense! Thanks for all your help today! :+1: :tada: :palm_tree:
